packages/sdk/src/types/ConnectorResponse.ts

```diff
@@ -38,7 +38,9 @@ export class ConnectorResponse<T> {
 
     public get result(): T {
         if (!this.isSuccess) {
-            throw new Error(`Can't get the value of an error Response. Use 'error' instead. Root error: ${this.error.code} - ${this.error.message}`);
+            throw new Error(
+                `Can't get the value of an error Response. Use 'error' instead. Root error: ${this.error.code} - ${this.error.stacktrace?.length ? this.error.stacktrace.join("\n") : this.error.message}`
```

Comment:
you hide the message if there is a stack trace

Reply:
Yea you might think so but JS Error stacktraces always contain the message also a bit weird

Reply:
IC